jwMaxwell/Spike-2
# Added Slisp and basic logging

**Related bugs/feature requests**


+ Logs file saves rather than chucking them in the console

**Describe the changes**

Added Slisp, the new and improved Spike-lisp

**Expected behavior**


**Screenshots**


**New npm dependencies**



**Testing Instructions**
Logging:
Send a bunch of messages and make sure they go to the log file

Slisp:
try these commands*;
$slisp
````
```lisp
(print (cons 'a '(b c d e)))
```
````
$slisp
````
```lisp
(print (+ 1 2 3 4 5))
```
````

$slisp
````
```lisp
(defun null? (x) (eq x '())) (null? '())
```
````

**Additional context**

Sorry for multiple changes in a single push. Not the best practice but whatever.
